BEIMA.DB.Schemas/device-schema.txt

```diff
@@ -1,6 +1,5 @@
 {
     "deviceTypeId": String,
-    "serialNumber": String,
```

Comment:
Why are we removing this?

Reply:
"serialNumber" is a duplicate value, as "serialNum" already exists.